tools/releaseBuild/images/Ubuntu16.04/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/powershell:ubuntu16.04
+FROM mcr.microsoft.com/powershell:ubuntu16.04
```

Comment:
Is it possible to verify the validity of this URL through a web UI? I don't have Docker setup anywhere, and going to mcr.microsoft.com redirects to a GitHub repository.

Reply:
It's not a url so no.

Reply:
![image](https://user-images.githubusercontent.com/10873629/118163851-4e6c6800-b3d7-11eb-95d0-8e74e8ec2657.png)
```suggestion
FROM mcr.microsoft.com/powershell:ubuntu-16.04
```

Reply:
Thanks 😂